### PR TITLE
Fix api cached pubkeys

### PR DESCRIPTION
**Pull request by maevyn11**
Prevented the app from unnecessarily waiting for the API to make a bunch of GET_PHONON_PUB_KEY calls when the keys are already cached. 

Also deleted or downgraded a bunch of pointless or obscure logs.

### CODE REVIEW

File: gui/api.go [hunk 1]
```diff
@@ -638,10 +649,12 @@ func (apiSession apiSession) listPhonons(w http.ResponseWriter, r *http.Request)
 	}
 
 	for _, p := range phonons {
-		p.PubKey, err = sess.GetPhononPubKey(p.KeyIndex, p.CurveType)
-		if err != nil {
-			http.Error(w, err.Error(), http.StatusInternalServerError)
-			return
+		if p.PubKey == nil {
```
Comment (npmaile) [inline gui/api.go:652]: good catch. I didn't even think about this or notice it.

**Review by npmaile — APPROVED** [review]

File: chain/ethChainService.go [hunk 2]
```diff
@@ -185,6 +190,8 @@ func (eth *EthChainService) calcRedemptionValue(balance *big.Int, gasPrice *big.
 	return valueMinusGas.Sub(balance, estimatedGasCost.Mul(gasPrice, big.NewInt(int64(gasLimit))))
 }
 
+func (eth *EthChainService) IsRedeemable()
```
Comment (npmaile) [inline chain/ethChainService.go:193]: no function body. oops

**Review by npmaile — CHANGES_REQUESTED** [review]

File: chain/ethChainService.go (hunk 2 above)
Comment (maevyn11) [reply chain/ethChainService.go:193]: oops. I moved that change to a separate branch, forgot to delete before committing this one. Will remove.

Comment (maevyn11) [reply chain/ethChainService.go]: https://github.com/GridPlus/phonon-client/pull/205/commits/580f2a21c8a451a1f576d62237ec6c145639d370

File: gui/api.go (hunk 1 above)
Comment (maevyn11) [reply gui/api.go:652]: neither did I until I was sitting waiting for the page to load while 15 GET_PHONON_PUB_KEYS ran :)

**Review by npmaile — APPROVED** [review]

**Pull request merged**